documentation/existing.md

```diff
@@ -158,7 +218,25 @@ Below is summary of supported parameters in the `vsphere.conf` file
 
         datastore = "DatastoreStorageFolder/datastore1"
 
-## Add flags to controller-manager, API server and Kubelet
+### Step - 6: Add flags to controller-manager, API server and Kubelet
+
+#### For kubernetes version 1.9.x
+
+** Master node **
+* Add following flags to kubelet running on master node and to the controller-manager and API server pods manifest files.
+
+```
+--cloud-provider=vsphere
+--cloud-config=<Path of the vsphere.conf file>
+```
+
+** Worker nodes **
```

Comment:
I still see ** Worker node ** in the documentation. Can you check it out.

Reply:
Done